aptos-move/framework/src/natives/cryptography/algebra/arithmetics/div.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright © Aptos Foundation
+
+use crate::{
+    abort_unless_arithmetics_enabled_for_structure, abort_unless_feature_flag_enabled,
+    natives::{
+        cryptography::algebra::{
+            abort_invariant_violated, feature_flag_from_structure, gas::GasParameters,
+            AlgebraContext, Structure, MOVE_ABORT_CODE_NOT_IMPLEMENTED,
+        },
+        helpers::{SafeNativeContext, SafeNativeError, SafeNativeResult},
+    },
+    safe_borrow_element, safely_pop_arg, store_element, structure_from_ty_arg,
+};
+use move_core_types::gas_algebra::NumArgs;
+use move_vm_types::{loaded_data::runtime_types::Type, values::Value};
+use num_traits::Zero;
+use smallvec::{smallvec, SmallVec};
+use std::{collections::VecDeque, ops::Div, rc::Rc};
+
+macro_rules! ark_div_internal {
```

Comment:
Shouldn't this be in `arithmetics/mod.rs` with everything else?

Reply:
The old `algebra/mod.rs` had 2k+ lines so I broke it into small pieces for easier navigation.
Any concerns?

Reply:
No concerns. Just noticed two `ark`-specific macros are declared in `aptos-move/framework/src/natives/cryptography/algebra/arithmetics/mod.rs` while the rest are declared next to the natives that use them.

I left comments thinking they should all be in that file above, but I guess it's fine because the two macros from the file above are shared by many natives.